### DISCUSSION TITLE
Unified Inbox search shows ALL Sent messages

### DISCUSSION THREAD

**Discussion by memphis-10** [protz/thunderbird-conversations#2075] [Q&A]
Hello,

Great plugin!  I have a question:

The unified inbox is showing threads as expected, however, it shows ALL sent messages.  How do I limit the search so the inbox shows inbox, and only sent replies (not the full sent folder)?

Thank you!

**Comment by Standard8** [comment]
This add-on doesn't change anything on the unified inbox. You need to ask Thunderbird about that - https://support.mozilla.org/thunderbird/

The add-on does show received and sent messages when showing the message previews. That's the whole point of the add-on and there's no way of only showing received.
